client/src/pages/NeedHelp.js

```diff
@@ -25,6 +26,9 @@ const INITIAL_STATE = {
   answers: [],
 };
 
+// for now just creating an object to collect the data that I need
+let stepsData = {};
```

Comment:
yeah so this is similar to the first version of the steps wizard flow that I made in the first week
because I wasn't sure how that library worked
but since then, I think `props.update()` was simply used to update the state the is kept in the parent component of the wizard, which is a simple solution to manage the state here
so at first sight, you kept that `props.update()` call but also added an object at the top of the file here which is probably not needed

Reply:
So I will be ignoring this for now since I will be only merging the Backend on this PR and wait for Cees to be done with his work for the filters, after that I will just make the api call from onsubmit() of these two pages.